components/pages/errorPageLayout/ErrorPageLayout.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from "react"
+
+const ErrorPageLayout = ({ lines }: { lines: (string | ReactNode)[] }) => {
+  return (
+    <div className="content-container flex items-center justify-center">
+      <div className="bg-white p-8 text-center shadow-md">
+        <h1 className="text-typo-heading-3 mb-6 font-bold">{lines[0]}</h1>
+        {lines.slice(1).map((line, index) => (
+          <p key={index} className="mb-6">
+            {line}
+          </p>
+        ))}
```

Comment:
I think this could be simpler and readable with using children prop and h1 as a title prop. 

```suggestion
        {children}
```

Reply:
We agreed to possibly create a future PR if we want the component structure to be changed